### PR TITLE
fix(FEV-758): Player v7 | Live 2.0.7| - Entry view - While the stream is in preview an error will be shown in the player

### PR DESCRIPTION
**Pull request by semarche-kaltura**
in decorator isLive method returns correct value for entry type

### CODE REVIEW

File: src/decorator/live-decorator.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export class KalturaLiveEngineDecorator implements KalturaPlayerTypes.IEngineDec
     }
 
     get active(): boolean {
-        return this._plugin.isLiveEntry();
+        return this._plugin.player.isLive();
```
Comment (yairans) [inline src/decorator/live-decorator.ts:30]: are there another uses of `_plugin.isLiveEntry` ? is so need to fix them as well and remove `_plugin.isLiveEntry` api and flag.
But first let's try to understand why the plugin didnt use the player api from the beginning. maybe you will find something in history

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/decorator/live-decorator.ts (hunk 1 above)
Comment (semarche-kaltura) [reply src/decorator/live-decorator.ts:30]: we use them in `src/middleware/live-middleware.ts`
but it seems like flag should be renamed to `isActive` that define if plugin already activated and added all the listeners.

Comment (semarche-kaltura) [reply src/decorator/live-decorator.ts:30]: PR

**Comment by yairans** [comment]
@semarche-kaltura in general please change the PR title to fix(FEV<ID>):<THE ISSUE (copy/paste from jira)>
then describe the change in the description block

File: src/decorator/live-decorator.ts (hunk 1 above)
Comment (yairans) [reply src/decorator/live-decorator.ts:30]: so in this issue the decorator will be active but the plugin not. does it make sense?

**Review by yairans — CHANGES_REQUESTED** [review]

**Comment by semarche-kaltura** [comment]
it should never happens.
We can add plugin listeners in decorator, but I think it's bad practice.

**Comment by semarche-kaltura** [comment]
Changes discussed with Yair Ansbacher.
Later we gonna review live plugin logic and will try avoid usage of boolean flags and use player api instead.

**Pull request merged**